# encoder_data.msg

float64 left_encoder_data
float64 right_encoder_data